templates/cassiopeia/scss/blocks/_footer.scss

```
@@ -8,11 +8,11 @@
 
   .grid-child {
     padding: 2.5rem ($cassiopeia-grid-gutter/2);
-  }
+    }
```

```suggestion
  }
```
Why are you doing unrelated and wrong code style changes?